src/utils/sqs_consumer.ts

```diff
@@ -95,10 +97,13 @@ export class SqsConsumer {
                 },
                 'Encountered error while handling message',
             );
-            // Retry message
-            await this._sqsClient.changeMessageVisibilityAsync(message.ReceiptHandle!, 0);
-            await delay(ONE_SECOND_MS);
-            return;
+
+            if (err instanceof SqsRetryableError) {
+                // Retry message
+                await this._sqsClient.changeMessageVisibilityAsync(message.ReceiptHandle!, 0);
```

Comment:
Could we add some logging just to be verbose?

Reply:
in this case, this pushes it back to the queue for any worker to grab, right?

Reply:
re logging: Yup - can do

re queue behavior: Yup - the message becomes immediately available for any worker to attempt next (i.e. doesn't go to the end of the queue, but stays in the front)